includes/frontend.php

```diff
@@ -1312,4 +1306,31 @@ private function process_more_tag( $content ) {
 
 		return force_balance_tags( $parts['main'] ) . $more_link;
 	}
+
+	private function is_optimized_assets_loading() {
+		return Plugin::$instance->experiments->is_feature_active( 'e_optimized_assets_loading' );
+	}
+
+	private function get_elementor_frontend_dependencies() {
+		$dependencies = [
+			'elementor-frontend-modules',
+			'elementor-dialog',
+			'elementor-waypoints',
+			'share-link',
+		];
+
+		if ( ! $this->is_optimized_assets_loading() ) {
+			wp_register_script(
```

Comment:
Can't it be part of the Assets Loader

Reply:
The frontend is enqueued before the assets loader is triggered (and the swiper is a dependency of the frontend).

Reply:
Fixed.